build.rs

```diff
@@ -3,34 +3,65 @@ extern crate serde;
 extern crate serde_json;
 
 use serde_json::Value;
+use std::env;
+use std::ffi::OsString;
 use std::fs::{self, File};
 use std::io::{BufReader, Write};
 
-#[derive(Deserialize, Debug)]
+#[derive(Serialize, Deserialize, Debug)]
 struct TestData {
     avro: Value,
     bigquery: Value,
     json: Value,
 }
 
-#[derive(Deserialize, Debug)]
+#[derive(Serialize, Deserialize, Debug)]
 struct TestCase {
     name: String,
     test: TestData,
```

Comment:
```suggestion
     test: TestData,
     #[serde(skip_serializing_if = "Option::is_none")]
     description: Option<Vec<String>>,
```

I think this should do the trick -- this will avoid serializing the field if it doesn't exist. This pattern gets used quite a bit in the rest of the project, actually.

Reply:
i used this method before I saw your suggestion, because it doesn't matter what type description is.
```suggestion
    test: TestData,
    #[serde(default)]
    #[serde(skip_serializing_if = "Value::is_null")]
    description: Value,
```